inspector/src/lod.ts

```diff
@@ -2,12 +2,17 @@ import { Effect } from "babylonjs/Materials/effect";
 
 let name = 'lodPixelShader';
 let shader = `
+#ifdef GL_ES
+precision highp float;
+#endif
+
 varying vec2 vUV;
 uniform sampler2D textureSampler;
 uniform float lod;
+uniform vec2 texSize;
 void main(void)
 {
-gl_FragColor=textureLod(textureSampler,vUV,lod);
+    gl_FragColor = textureLod(textureSampler,vUV,lod);
```

Comment:
this requires an extension in Webgl1 with smthg like #extension GL_EXT_shader_texture_lod : enable

Reply:
and if texture lod are not supported, you could disable the mip map tool :-)

Reply:
Added a check for that! Thanks :)